gammapy/utils/tests/test_time.py

```diff
@@ -0,0 +1,27 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from numpy.testing import assert_almost_equal
+from astropy.time import Time, TimeDelta
+from ..time import time_ref_from_dict, time_relative_to_ref
+
+
+def test_time_ref_from_dict():
+    mjd_int = 500
+    mjd_frac = 0.5
+    time_ref_dict = dict(MJDREFI=mjd_int, MJDREFF=mjd_frac)
+    time_ref = time_ref_from_dict(time_ref_dict)
+    decimal = 4
+    assert_almost_equal(time_ref.mjd, mjd_int + mjd_frac, decimal)
+
+
+def test_time_relative_to_ref():
+    mjd_int = 500
```

Comment:
Again, the lines where you set `mjd_int`, `mjd_frac` and `decimal` below can be removed, just pass the values directly as keyword arguments.

The main reasons to define temp variables are:
* You need the variable in many places in a piece of code
* Expressions become too long and need to be split up in pieces
* You pass values in somewhere in a function where you can only use positional arguments (i.e. have to use an un-pythonic, bad API) and not giving it a name reads to cryptic code (like you had for word / pix coordinate conversions).